templates/terraform/extra_schema_entry/forwarding_rule.erb

```diff
@@ -0,0 +1,21 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2019 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+"ip_version": {
+	Type:         schema.TypeString,
+	Optional:     true,
+	Removed:      "ipVersion is not used for regional forwarding rules. Please remove this field if you are using it.",
```

Comment:
Do we need the rest of the information (ValidateFunc, ForceNew) if this field is marked removed? Does removed actually prevent the user from specifying it?